src/main/java/org/noteskeeper/server/IGreetingService.java

```diff
@@ -0,0 +1,5 @@
+package org.noteskeeper.server;
+
+public interface IGreetingService {
```

Comment:
It is not a variable/property name, but name of a type (class/interface). So it is not really associative with plural form. But as I said earlier, it is just a suggestion. If you like *Impl more, let's use it.